src/express/incoming/manager.ts

```diff
@@ -0,0 +1,51 @@
+import * as mongoose from 'mongoose';
+import { IIncomingFile, INewIncomingFile } from './interface';
+import IncomingFileModel from './model';
+
+/**
+ * Create a new feature.
+ * @param {INewFeature} feature - The feature to create.
+ * @returns {Promise<IFeature>} - Promise object containing the created feature.
+ */
+
+/**
+ * Create a new incoming file
+ * @param {INewIncomingFile} file - the file to create
+ * @returns {IIncomingFile} - Promise containing the incoming file
+ */
+export const createIncomingFile = (file: INewIncomingFile): Promise<IIncomingFile> => {
+    return IncomingFileModel.create(file);
+};
+
+/**
+ * Get incoming file.
+ * @param  fileId - id of current file .
+ * @returns {Promise<IIncomingFile>} - Promise object containing the file with current id.
+ */
+export const getIncomingFileById = async (fileId: mongoose.Types.ObjectId): Promise<IIncomingFile> => {
+    const result = await IncomingFileModel.findOne({ _id: fileId }).exec();
+    if (!result) throw new Error('Incoming file not found');
+    return result;
+};
+
+/**
+ * Get incoming file.
+ * @param {Partial<IIncomingFile>} query - The query to filter the files.
+ * @returns {Promise<IIncomingFile[]>} - Promise object containing the filtered files.
+ */
+
+export const getMultipleFiles = async (query: any): Promise<IIncomingFile[]> => {
```

Comment:
Why not name the function `getIncomingFiles`?

```suggestion
export const getIncomingFiles = async (query: any): Promise<IIncomingFile[]> => {
```